### ISSUE TITLE
Boosted OMM UI

### ISSUE THREAD

**Issue by parrot9design** [ommfinance/openmoneymarket-frontend#72]
Here are the UI changes required for the Boosted OMM functionality. You can preview the main functionality in the [development prototype](http://omm-development.s3-website-us-east-1.amazonaws.com/demo/rewards/). See [this pull request](https://github.com/parrot9design/Omm/pull/27) for the core code changes. I mocked up the onboarding states while writing out the details for this issue, but we can provide more guidance for the code if needed.

## Empty states

**Rewards page**
<img width="944" alt="Screen Shot 2022-03-11 at 8 34 17 AM" src="https://user-images.githubusercontent.com/63842577/157741544-63e165eb-3a4f-4805-9777-1e27aa566831.png">

- Show `1 x` under market and/or liquidity rewards if they've participated, otherwise show a `–`.

**Vote page**
<img width="953" alt="Screen Shot 2022-02-22 at 7 30 20 PM" src="https://user-images.githubusercontent.com/63842577/155076273-d4df86d1-d4fb-47e0-a62c-6444ca8ba1ba.png">
- Hide the voting power until they've locked up OMM.

---

## With unlocked OMM

**Rewards page**
<img width="946" alt="Screen Shot 2022-03-11 at 8 39 49 AM" src="https://user-images.githubusercontent.com/63842577/157741720-63dbd9c6-f0e5-4f8b-a607-4ed98b1e7ea8.png">

**Vote page**
<img width="961" alt="Screen Shot 2022-02-22 at 7 45 43 PM" src="https://user-images.githubusercontent.com/63842577/155077471-da4c7c4b-1fef-4e43-bcd3-213c213b0cd7.png">

---

## Edit mode

**Vote page**
<img width="949" alt="Screen Shot 2022-02-22 at 7 51 36 PM" src="https://user-images.githubusercontent.com/63842577/155078497-6b14ba60-f21f-48ca-b85a-1ebbcf5a9aae.png">
<img width="973" alt="Screen Shot 2022-02-22 at 7 51 46 PM" src="https://user-images.githubusercontent.com/63842577/155078519-4e7ba354-8dad-4aeb-ab87-b37bab9aa7cb.png">

- Time period: After selecting a time period, the dropdown should close and the `Locked until` date should update. Default to 1 week.
- `Your voting power` and the bOMM values should update as you drag the slider / when you update the text box. 

**Rewards page**
The Rewards version functions the same way, with some variance.

When you click `Lock up OMM`, it should initially look like this (assuming participation in a single market and LP):
<img width="957" alt="Screen Shot 2022-03-11 at 8 47 17 AM" src="https://user-images.githubusercontent.com/63842577/157742921-d3dced96-95a0-414f-9f80-b8861177d5db.png">

As you move the slider, the values should change:
<img width="945" alt="Screen Shot 2022-03-11 at 8 33 36 AM" src="https://user-images.githubusercontent.com/63842577/157742027-3dcb1695-b0b8-40e5-b8a8-a5b016a011b7.png">

- This nwo assumes you've participated in multiple markets or liquidity pools, otherwise it should show a single number. The maximum (at least initially) will be set at 2.5x.

---

## Confirmation modal

<img width="348" alt="Screen Shot 2022-02-22 at 7 57 25 PM" src="https://user-images.githubusercontent.com/63842577/155078956-a75ab9ba-0b4a-4ac9-8e6e-baf7322bb256.png">

**Banner messages**

> Locking up Omm Tokens...

> 2,000 OMM locked until 22 August, 2022.

---

## With locked OMM

**Rewards page**

<img width="948" alt="Screen Shot 2022-03-11 at 8 33 57 AM" src="https://user-images.githubusercontent.com/63842577/157743650-26cbe5cf-17fc-4e4e-9838-7ada7ae20db6.png">

**Vote page**

<img width="838" alt="Screen Shot 2022-03-01 at 10 08 19 AM" src="https://user-images.githubusercontent.com/63842577/156058943-d0968849-c5c7-4530-8bc4-9b53570bfe1f.png">

---

## Additional changes

The next time you go to adjust your locked OMM, you'll see that the amount already in there is locked until the time is up:
<img width="954" alt="Screen Shot 2022-02-22 at 7 57 54 PM" src="https://user-images.githubusercontent.com/63842577/155079069-ed9e98ae-d55a-4ae8-b5fc-661735e6e660.png">

You can increase the amount that's locked for the existing time (i.e. until 20 August 2022), or use the dropdown to increase the time period. We should hide the smaller time frames so people can't game the system:

<img width="1003" alt="Screen Shot 2022-02-22 at 7 27 07 PM" src="https://user-images.githubusercontent.com/63842577/155079297-f2b7057a-d376-4e19-a771-8e4e3df4cbfc.png">

**Comment by parrot9design** [comment]
I've pushed an update to the code to include the empty states and views without locked OMM. [Commit details here](https://github.com/parrot9design/Omm/pull/27/commits/382e0d88202e11af1fe2b7bd40617b2aa1c26a1c)

I had to comment out `<div id="slider-boost" disabled=""></div>` for the unlocked OMM states to prevent issues with the id selector, but the slider should appear and be empty in this state.

We may need to push another update to show ranges for market and liquidity APY/APRs, as well as the APY for Boosted OMM rewards, but will wait until we hear how the devs plan to handle the API calls for this.
